CHANGELOG.md

```diff
@@ -381,4 +388,4 @@
 
 ## v0.0.1 - 2022-06-21
 ### Added
-* Initial release
+* Initial release
```

Comment:
Unnecessary noise?

Reply:
Yes, sorry, it's very likely to be related to my changes. Maybe we should upgrade the version of `changie` to match the latest one?